class Failure {
  final String message;
  Failure([
    this.message = 'Sorry, an unexpected error has occured',
  ]);
}
